### PR TITLE
Delete limit number of maps shows when deleting a dataset

### PR DESCRIPTION
**Pull request by elenatorro**
Related issue: https://github.com/CartoDB/cartodb/issues/14685

Context:
=== 

The modal warning with the list of maps that would be affected if a dataset is deleted shows the maps list without a wrong space bottom and a maximum of 3 maps.

Needs to be reviewed by design cc @CartoDB/design-team 
Screenshot 2019-05-09 at 12 26 57

Screenshot 2019-05-09 at 12 22 17

Acceptance
===

This PR:

- Adds a bottom space to map list elements
- Adds an overflow to the map list container to allow having more than 3 maps (which was the default max number of possible affected maps shown). I've changed this number to allow the user to visualize **all** the possible affected maps, but we can go back to show a maximum of 3 maps.

staging: https://modal-style.carto-staging.com
user: modal-style

### CODE REVIEW

File: assets/stylesheets/common/maps-list.scss [hunk 1]
```diff
@@ -12,9 +12,11 @@
   align-items: center;
   justify-content: flex-start;
   width: $sLayout-width + $sMargin-element;
+  height: $sDialog-Height;
```
Comment (alejandraarri) [inline assets/stylesheets/common/maps-list.scss]: I would change this to a max-height, so when there is only one map content stays centered as the original design.

File: assets/stylesheets/variables/sizes.scss [hunk 2]
```diff
@@ -85,3 +85,6 @@ $sMedia-xxl: 1600px;
 
 $sMedia-min-w-vertical: 600px;
 $sMedia-max-w-vertical: 759px;
+
+// Dialog
+$sDialog-Height: 600px;
```
Comment (alejandraarri) [inline assets/stylesheets/variables/sizes.scss]: Maybe the variable could be something like `$sDialogBody-Height` or `$sDialogInner-Height` so it is not confused with the total dialog height?

File: lib/assets/javascripts/dashboard/views/dashboard/dialogs/delete-items/delete-items.tpl [hunk 3]
```diff
@@ -24,7 +24,7 @@
 <% if (affectedVisCount > 0) { %>
   <ul class="Dialog-body MapsList MapsList--centerItems is-singleRow">
```
Comment (alejandraarri) [inline lib/assets/javascripts/dashboard/views/dashboard/dialogs/delete-items/delete-items.tpl]: We have utility classes to fix vertical spacing easily. In this case, you need `u-pt--0 u-pb--20` to counteract the padding defined in MapsList.

File: assets/stylesheets/variables/sizes.scss (hunk 2 above)
Comment (elenatorro) [reply assets/stylesheets/variables/sizes.scss]: Totally! I'd go with `$sDialogInner-Heigh` 👍

**Comment by elenatorro** [comment]
Changes applied, re-deploying to staging 👍

**Comment by alejandraarri** [comment]
Acceptance done! Works like a charm!

File: assets/stylesheets/variables/sizes.scss [hunk 4]
```diff
@@ -87,4 +87,4 @@ $sMedia-min-w-vertical: 600px;
 $sMedia-max-w-vertical: 759px;
 
 // Dialog
-$sDialogInner-Height: 600px;
\ No newline at end of file
+$sDialogInner-Height: 480px;
```
Comment (alejandraarri) [inline assets/stylesheets/variables/sizes.scss]: I'm thinking that this should be something like `calc(100vh - 350px);` so we make sure the header + footer + share message = 350px always appear within the dialog.

File: assets/stylesheets/common/utilities.scss [hunk 5]
```diff
@@ -154,6 +154,10 @@
   text-transform: uppercase;
 }
 
+.u-border-bottom {
```
Comment (alejandraarri) [inline assets/stylesheets/common/utilities.scss]: Nice touch! 👌

File: assets/stylesheets/variables/sizes.scss (hunk 4 above)
Comment (elenatorro) [reply assets/stylesheets/variables/sizes.scss]: Yes! you're 100% right. After testing it with several sizes and with the shared datasets warning, used `calc(100vh - 430px)` to add also the warning size (80px), so it looks now like this:

Screenshot 2019-05-10 at 08 16 09

File: assets/stylesheets/common/utilities.scss (hunk 5 above)
Comment (elenatorro) [reply assets/stylesheets/common/utilities.scss]: Nice catch! Didn't think about it, fixed 👍

**Pull request merged**